RadarSDK/Radar.m

```diff
@@ -1036,6 +1038,19 @@ + (void)setLogLevel:(RadarLogLevel)level {
     [RadarSettings setLogLevel:level];
 }
 
++ (void)writeLocalLog:(NSString *)message {
+    [[RadarLogger sharedInstance] logWithLevelLocal:RadarLogLevelInfo type:RadarLogTypeNone message:message];
+
+}
+
++ (void) logUserTermination {
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    [dateFormatter setDateFormat:@"yyyy-MM-dd HH:mm:ss"];
+    NSString *dateString = [dateFormatter stringFromDate:[NSDate date]];
+    NSString *message = [NSString stringWithFormat:@"User terminated app at %@", dateString];
```

Comment:
Let's change this to `App terminated at %@"` — we don't know affirmatively whether or not it's a user driven termination